public/claimlist.php

```diff
@@ -10,7 +10,10 @@
 use Illuminate\Support\Facades\Blade;
 use Illuminate\Support\Str;
 
-authenticateFromCookie($user, $permissions, $userDetails);
+if (!authenticateFromCookie($user, $permissions, $userDetails, Permissions::Registered)) {
+    header("Location: /login");
+    exit;
```

Comment:
This should probably just be:
```
return redirect(route('login'));
```

There's probably even a way to pass it the current page so it'll come back after login.

Reply:
Changed to use a `redirect()` call in latest, however I'm not able to seem to get the `intended_url` value to work correctly.

Reply:
It looks like `redirect()` automatically does that. After pasting the URL into an incognito window, getting redirected to the login page, and entering credentials, it took me back to the pasted URL.